sankalparorame.md

```diff
@@ -0,0 +1,28 @@
+---
+name: "@sankalparorame"
+project: "TheMediBand"
+---
+
+# Project Name
+TheMediBand
+
+## Summary
+I am going to build a medical device that combines a number of devices in form of a compact and easy to use neckbandand also costs far less than all of them combined. It is aimed at overcoming the problem of delay in diagnosis and treatment of Cardiovascular Diseases (CVDs) due to lack of equipment, specially in rural areas in India. The device combines ECG, EEG and EKG modules alongside being a defibrillator for people undergoing cardiac arrest. It also comprises of a peizo modem that shall help in ascerating blood flow related issues including congestions.
+
+## Plan
+Firstly, shall connect all sensors to Pi and program them to make them functioning and cast data to app. Later once the aimed efficiency is achieved though trial error and program refinement, I shall design a neckband case to enclose all the sensors within it to match the design and finally work on PC Client and mobile app and their connectivity.
```

Comment:
Do you have a Pi to use for this?

I noticed some items in your budget that I'm not sure how they are used in your project. Could you describe how you will use those items here? In particular, the Arduino. 

Also what programming languages or frameworks do you plan to use to program this?

How do you plan to design the case and what will it be made of?

Reply:
@sankalparorame: did you get a chance to address these questions? I saw that you requested another review but not sure if I missed something!